tests/release/e2e-test-default-with-deployment.go

```diff
@@ -0,0 +1,231 @@
+		},
+	}
+
+	BeforeAll(func() {
+		kubeController := tekton.KubeController{
+			Commonctrl: *framework.CommonController,
+			Tektonctrl: *framework.TektonController,
+		}
+
+		_, err := framework.CommonController.CreateTestNamespace(devNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating devNamespace: %v", err)
+		klog.Info("Dev Namespace created: ", devNamespace)
+
+		_, err = framework.CommonController.CreateTestNamespace(managedNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating managedNamespace: %v", err)
+		klog.Info("Managed Namespace created: ", managedNamespace)
+
+		klog.Infof("Wait until the 'pipeline' SA is created in %s namespace \n", managedNamespace)
+		Eventually(func() bool {
+			sa, err := framework.CommonController.GetServiceAccount("pipeline", managedNamespace)
+			return sa != nil && err == nil
+		}, pipelineServiceAccountCreationTimeout, defaultInterval).Should(BeTrue(), "timed out when waiting for the \"pipeline\" SA to be created")
+
+		sourceAuthJson := utils.GetEnv("QUAY_TOKEN", "")
+		Expect(sourceAuthJson).ToNot(BeEmpty())
+
+		destinationAuthJson := utils.GetEnv(constants.QUAY_OAUTH_TOKEN_RELEASE_DESTINATION, "")
+		Expect(destinationAuthJson).ToNot(BeEmpty())
+
+		_, err = framework.CommonController.CreateRegistryAuthSecret(hacbsReleaseTestsTokenSecret, devNamespace, sourceAuthJson)
+		Expect(err).ToNot(HaveOccurred())
+
+		_, err = framework.ReleaseController.CreateRegistryJsonSecret(redhatAppstudioUserSecret, managedNamespace, destinationAuthJson, destinationKeyName)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = framework.CommonController.LinkSecretToServiceAccount(devNamespace, hacbsReleaseTestsTokenSecret, "pipeline")
+		Expect(err).ToNot(HaveOccurred())
+
+		klog.Info("Dev Namespace:", devNamespace)
+		klog.Info("Managed Namespace:", managedNamespace)
+
+		// Copy the public key from tekton-chains/signing-secrets to a new
+		// secret that contains just the public key to ensure that access
+		// to password and private key are not needed.
+		publicKey, err := kubeController.GetPublicKey("signing-secrets", constants.TEKTON_CHAINS_NS)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(kubeController.CreateOrUpdateSigningSecret(
+			publicKey, publicSecretNameAuth, managedNamespace)).To(Succeed())
+
+		defaultEcPolicy = ecp.EnterpriseContractPolicySpec{
+			Description: "Red Hat's enterprise requirements",
+			PublicKey:   string(publicKey),
+			Sources: []ecp.Source{
+				{
+					Name: "ec-policies",
+					Policy: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//policy",
+					},
+					Data: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//data",
+					},
+				},
+			},
+			Exceptions: &ecp.EnterpriseContractPolicyExceptions{
+				NonBlocking: []string{"tasks", "attestation_task_bundle", "java", "test", "not_useful"},
+			},
+		}
+
+		_, err = framework.CommonController.CreateConfigMap(cm, devNamespace)
```

Comment:
as mentioned previously, no need to create the config map, because it's not used

Reply:
removed